tests/unit/Trace/Reporter/ZipkinReporterTest.php

```diff
@@ -77,4 +78,30 @@ public function testFormatsTrace()
             }
         }
     }
+
+    public function testSpanKind()
```

Comment:
I was planning on writing a ZipkinV2Reporter with #20.
I'm not sure what the future status of the v1 API is, or how many people use it vs v2